lang/java/reef-common/src/main/java/org/apache/reef/runtime/common/files/REEFFileNames.java

```diff
@@ -222,19 +223,31 @@ public String getDriverHttpEndpoint() {
 
   /**
    * @return File name that contains the security token identifier
+   * @deprecated - use getSecurityTokensFile()
+   * TODO[JIRA REEF-1887]: Remove this method.
    */
   public String getSecurityTokenIdentifierFile() {
     return SECURITY_TOKEN_IDENTIFIER_FILE;
   }
 
   /**
    * @return File name that contains the security token password
+   * @deprecated - use getSecurityTokensFile()
+   * TODO[JIRA REEF-1887]: Remove this method.
    */
   public String getSecurityTokenPasswordFile() {
     return SECURITY_TOKEN_PASSWORD_FILE;
   }
 
   /**
+   * @return Name of the file that contains security tokens and passwords.
+   * This call supersedes getSecurityTokenIdentifierFile() and getSecurityTokenPasswordFile().
```

Comment:
add TODO tag so we can remove it later
use our standard format for `@deprecated` tags:
```java
// TODO[JIRA REEF-1887] This call supersedes getSecurityTokenIdentifierFile() and getSecurityTokenPasswordFile(). Remove this comment line in 0.17
```